Makefile.org

```diff
@@ -685,7 +685,7 @@ install_docs:
 		(cd `$(PERL) util/dirname.pl $$i`; \
 		pod2man \
 			--section=$$sec --center=OpenSSL \
-			--release=$(VERSION) `basename $$i`) \
+			--release="OpenSSL $(VERSION)" `basename $$i`) \
```

Comment:
Consistency with other man pages. Most of the other ones I've seen have either the package name and version number, or just the package name. I can revert it if you prefer just version.